### PR TITLE
docs(README): Added documentation on how to run the bot

### PR DESCRIPTION
**Pull request by MrAuro**
## Description
Adds documentation for obtaining the needed keys and running the bot

## Types of changes
- Documentation (non-breaking change which updates documentation)

## Checklist

- [X] Changes follow the CONTRIBUTING.md guidelines.
- [X] Update necessary documentation accordingly.
- [X] Check issues and pull requests first. You don't want to duplicate effort.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
 💰 A bot that uses Uberduck AI to make bit donations have an AI voice.
 
+### How to run
+
+1. Clone the repo
+2. Obtain the following keys:
+
+    1. [An Uberduck API key and secret](https://uberduck.ai/account/manage)
+    2. [A Twitch Client Id and secret](https://dev.twitch.tv/console/apps/create)
+        - Add an OAuth Redirect URI of `http://localhost:3000/`
```
Comment (MrAuro) [inline README.md:12]: Not sure if this is the correct port, lmk if it is something different

Comment (mmattbtw) [reply README.md:12]: Corrected the port (really weird port lol): 
```suggestion
        - Add an OAuth Redirect URI of `http://localhost:17563/`
```

**Review by mmattbtw — CHANGES_REQUESTED** [review]

**Review by mmattbtw — APPROVED** [review]
lgtm! :)

**Comment by mmattbtw** [comment]
@all-contributors add @MrAuro for doc, and code

**Pull request merged**